Stripe/STPPaymentContext.m

```diff
@@ -598,15 +598,14 @@ three_d_secure types of required or optional (not_supported are filtered earlier
                                                                                currency:self.paymentCurrency
                                                                               returnURL:self.configuration.returnURLBlock().absoluteString
                                                                                    card:cardSource.stripeID];
+    threeDSecureParams.metadata = self.sourceInformation.metadata;
 
     STPSourceCompletionBlock threeDSSourceCompletion = ^(STPSource * _Nullable threeDSSource, NSError * _Nullable error) {
         if (error
             || threeDSSource == nil
             || threeDSSource.flow != STPSourceFlowRedirect) {
             if (!threeDSecureRequired
-                && error.domain == StripeDomain
-                && error.code == STPCardError
-                && [error.userInfo[STPCardErrorCodeKey] isEqualToString:STPPaymentMethodNotAvailable]) {
+                && [error.userInfo[STPStripeErrorCodeKey] isEqualToString:@"payment_method_not_available"]) {
```

Comment:
I'm fine with not putting it under card_error I guess. If you want to just define an NSString const that is that expected error though? Not 100% sure of the Swift bridging implications of it being in that enumeration or not though.